lib/ansible/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -133,13 +155,17 @@ def connect_to_influxdb(module):
     username = module.params['username']
     password = module.params['password']
     database_name = module.params['database_name']
+    ssl = module.params['ssl']
+    validate_certs = module.params['validate_certs']
 
     client = InfluxDBClient(
         host=hostname,
         port=port,
         username=username,
         password=password,
-        database=database_name
+        database=database_name,
+        ssl=ssl,
+        validate_certs=validate_certs
```

Comment:
this is not quite correct, should be `verify_ssl=validate_certs`

Reply:
thanks for pointing this out.